src/Docker/Client/Types.hs

```diff
@@ -643,7 +645,7 @@ defaultContainerConfig imageName = ContainerConfig {
                      , cmd=[]
                      , volumes=[]
                      , workingDir=Nothing
-                     , entrypoint=Nothing
+                     , entrypoint=NoEntrypoint
```

Comment:
The only thing that comes to mind is will this correctly override an entrypoint of and existing image. Ie. if we wan't to change the entrypoint of and image to null, effectively stripping the image of it's entrypoint (for some reason) ... I'm not actually sure this is possible with docker at all. Also is this a sane default?